### ISSUE TITLE
Request for  Baseline Training Details on Electricity Dataset

### ISSUE THREAD

**Issue by Amitica** [ameliawong1996/From_News_to_Forecast#9]
Thank you for your outstanding work! I am reproducing the Electricity dataset on the Time Series Library benchmark and found that the UniModal model PatchTST performs significantly worse than the results reported in the paper. Could the authors provide the training details for the baseline?

**Comment by JasonGUTU** [comment]
Directly running the code in the Time Series Library benchmark yields suboptimal results. We searched for hyperparameters, mainly optimizing learning rate, batch size, etc. After optimization, the baseline results were significantly improved.

**Comment by Amitica** [comment]
I have tried various hyperparameter adjustments but found that the performance of the Baseline model could never reach the results mentioned in the paper. I extracted data from the weather_load.csv file in the raw time series data of the GitHub repository, covering the period from 2019-01-01 to 2021-07-25. The target variable is TOTALDEMAND. I applied numerical encoding to State and set the following parameters:
	•	freq = 30min
	•	features = 'MS'
	•	seq_len = 48, label_len = 48, pred_len = 48
	•	enc_in = 11, dec_in = 1, c_out = 1
	•	train_epochs = 3

I searched for learning_rate in [0.001, 0.0005, 0.0001, 0.00005] and batch_size in [32, 64, 128], but the performance remained suboptimal. For example, the best performance of PatchTST was consistently around 324, which is far from the expected results.

Could you provide detailed Baseline training configurations and settings?

**Comment by JasonGUTU** [comment]
There are a few issues to note.

- First, Your learning rate might be too low. Have you noticed any patterns among the parameters you've experimented with? Does a larger learning rate lead to better performance? In some cases, the optimal learning rate can be as high as 0.1. Depending on the optimizer you are using, you might need to explore a wider range of learning rates.
- Second, Is the data you are using the same as our test data? Does it include a lot of additional supplementary information? Please note that the testing dates we use are provided in our JSON training and test datasets.
